dash/include/dash/util/LocalityDomain.h

```diff
@@ -133,10 +105,43 @@ class LocalityDomain
 
   }; // class LocalityDomain::iterator
 
+}
+
+/**
+ * Wrapper of a single \c dart_domain_locality_t object.
+ *
+ * Usage examples:
+ *
+ * \code
+ * dash::util::TeamLocality     team_locality(dash::Team::All());
+ * dash::util::LocalityDomain & domain = team_locality.domain();
+ *
+ * // Leader unit in second subdomain:
+ * dart_unit_t leader_id = domain[1].leader_unit();
```

Comment:
Ah, yes. Will grep for `dart_unit_t` in `/dash`, it should probably never be used there.